Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.Events.cs

```diff
@@ -12,11 +12,6 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// </summary>
     public partial class TokenizingTextBox : Control
     {
-        /// <summary>
```

Comment:
See below, we still need this event.

Reply:
We may also want to name it `TextChanged` to align with the base AutoSuggestBox event name.